docs/access-management/rbac.md

```diff
@@ -0,0 +1,54 @@
+---
+products: oss-enterprise, cloud-teams
+---
+
+# Role Based Access Control (RBAC)
+
+Role Based Access Control allows a user with Administrative access to apply roles to users, granting different levels of permission within an organization or workspace. 
```

Comment:
We should include some screenshots on how to provide workspace permissions, which is less straightforward and frequently leads to user confusion.

Reply:
Is there a test org I can use to do screenshots? In the workspace I can access, it doesn't seem like I can set it up the way I'd expect to see in a product screenshot.